java/arcs/core/storage/database/DatabaseManifest.kt

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+package arcs.core.storage.database
+
+/**
+ * Mutable variant of [DatabaseManifest], which allows the user to register newly-created databases.
```

Comment:
The term Manifest has a meaning in Arcs, it could generate confusion, I think a better name for this would be DatabaseRegistry, wdyt?

Reply:
I agree, DatabaseRegistry SGTM

Reply:
Good call, done!